deployment.rst

```diff
@@ -54,7 +54,8 @@ Using Source Control
 If you're using source control (e.g. Git or SVN), you can simplify by having
 your live installation also be a copy of your repository. When you're ready
 to upgrade it is as simple as fetching the latest updates from your source
-control system.
+control system. A common approach for this in Git is, to tag your release and
+check out the Git tag on deployment (see `Git Tagging`_).
```

Comment:
@dominikhajduk Checking out the master branch is safer. Since you can rollback more clear to a previous release tag in case of failure. Git tags are the standard in module releases in the Drupal project.